modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Options/Options.es.js

```diff
@@ -380,7 +380,14 @@ class Options extends Component {
 		if (defaultLanguageId === editingLanguageId) {
 			const copyLanguageLabels = (languageId, options) => {
 				return options.map(({label, value}, index) => {
-					const option = newValue[languageId][index];
+					let option = newValue[languageId][index];
+
+					if (!option) {
```

Comment:
@carolmariaabb, what do you think to use the [Double not operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators#Logical_NOT) here instead of the logical not operator?

Reply:
I believe is a better alternative @natocesarrego, thanks!